vllm/v1/attention/backends/mla/common.py

```diff
@@ -631,6 +656,13 @@ def build(self,
             split_decodes_and_prefills(common_attn_metadata,
                                        decode_threshold=self.reorder_batch_threshold)
 
+        # Note(hc): update seq_lens of decode reqs under DCP.
+        if self.dcp_world_size > 1:
+            seq_lens[:num_decodes] = seq_lens[:num_decodes] \
+                // self.dcp_world_size + \
+                (get_dcp_group().rank_in_group <= (seq_lens[:num_decodes] - 1) \
```

Comment:
keep `get_dcp_group().rank_in_group` during init to avoid runtime call? similar to `self.dcp_world_size`